goose3/extractors/authors.py

```diff
@@ -70,7 +69,9 @@ def __get_authors_from_schema(self):
                 schema_authors = [schema_authors]
             for author in schema_authors:
                 if isinstance(author, dict):
-                    authors.append(author.get("name", ""))
+                    author = author.get("name", None)
+                    if author:
```

Comment:
please use a different variable than author, it is the same as the loop variable `extracted_author` may be more appropriate